test/functionalTest/cases/000_bad_requests/pagination_error_in_uri_params.test

```diff
@@ -37,7 +37,7 @@ echo
 echo
 
 echo "+++++ 2. bad characters in URI param 'offset'"
-url="/v1/queryContext?limit=0&offset=qwe"
+url="/v1/queryContext?offset=qwe"
```

Comment:
In this functest the idea is to get a complaint about offset 'awe'.
If I leave the 'offset=0', then it complains about offset instead.
I don't know why this has changed, but the easiest way to get it to complain as 'wanted', was to remove limit=0.
Why didn't the broker complain about limit=0 before?  Beats me ...